modals/request.py

```diff
@@ -51,4 +51,13 @@ async def on_submit(self, interaction: discord.Interaction):
             return
 
         # Send the user's request to the moderator-only channel test
-        await channel.send(f"User Requesterssssss: \n\n{self.request.value}")
+        await channel.send(f"User Request: \n\n{self.request.value}")
+        msg = Embed(
+            title="Request",
+            description=f"Heads up! {__name__} would like to learn about: \n\n{self.request.value}",
+            color=0xff6700
```

Comment:
I think they use "colour" for color.  The hex code has to be converted to a string.